plc4j/drivers/opcua/src/main/java/org/apache/plc4x/java/opcua/protocol/OpcuaField.java

```diff
@@ -68,12 +69,14 @@ public static OpcuaField of(String address) {
         String identifier = matcher.group("identifier");
 
         String identifierTypeString = matcher.group("identifierType");
-        OpcuaIdentifierType identifierType = OpcuaIdentifierType.fromString(identifierTypeString);
+        OpcuaIdentifierType identifierType = OpcuaIdentifierType.enumForValue(identifierTypeString);
```

Comment:
What happens if it is an invalid value?

Reply:
A null pointer exception is eventually raised, might be worth adding a contains key to the enum template as it looks like it is a concern with all of the Field classes.